neptune/__init__.py

```diff
@@ -26,15 +26,15 @@
 __lock = threading.RLock()
 
 
-def init(project_qualified_name=None, api_token=None):
+def init(project_qualified_name=None, api_token=None, proxies={}):
```

Comment:
Please, go with `proxies=None` and in the method body do
```python
if proxies is None:
    proxies = {}
```

Using a dictionary as default parameter value is a [no-no](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html) in Python :)

Reply:
I put instead under Client.__init__():
```python
if proxies is not None:
    self._update_proxies()
```